Sources/ProjectDescription/DeploymentTarget.swift

```diff
@@ -12,18 +12,22 @@ public enum DeploymentTarget: Codable, Hashable {
     case watchOS(targetVersion: String)
     /// The minimum tvOS version your product will support.
     case tvOS(targetVersion: String)
+    /// The minimum visionOS version your product will support.
+    case visionOS(targetVersion: String)
```

Comment:
I think it should be xrOS?

Reply:
Apple refers to it as visionOS everywhere user-facing:
<img width="317" alt="image" src="https://github.com/tuist/tuist/assets/6877780/9481aa23-c21f-4930-a559-ae88c9eb3e47">